### PR TITLE
reset css

### PR DESCRIPTION
**Pull request by AndrzejPec**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import Image from 'next/image';
 import styles from './page.module.css';
+import '../styles/reset.css';
```
Comment (MateuszSlezak) [inline src/app/page.tsx:3]: that should be in layout.tsx before import './globals.scss';

**Review by MateuszSlezak — CHANGES_REQUESTED** [review]
.

**Review by MateuszSlezak — CHANGES_REQUESTED** [review]
.

File: src/app/page.tsx (hunk 1 above)
Comment (AndrzejPec) [reply src/app/page.tsx:3]: Indeed it should. I'm on it :)

**Pull request closed without merging**